juice-examples/src/main.rs

```diff
@@ -671,7 +679,7 @@ fn get_regr_iter() -> impl Iterator<Item = (f32, Vec<f32>)> {
 }
 
 fn get_packed_regr_iter() -> impl Iterator<Item = (f32, Vec<Vec<f32>>)> {
-    let rdr = Reader::from_reader(File::open("assets/normalised_mackeyglass_lstm.csv").unwrap());
```

Comment:
I've reverted this since - lstm_equiv was a test file that I was using to compare this performance against keras.